site/content/en/docs/Advanced/controlling-disruption.md

```diff
@@ -72,6 +72,9 @@ GKE Autopilot supports only `Never` and `Always`, not `OnUpgrade`.
 
 Outside of Cluster Autoscaler, the main source of disruption for long sessions is node upgrade. On some cloud products, such as GKE Standard, node upgrades are entirely within your control. On others, such as GKE Autopilot, node upgrade is automatic. Typical node upgrades use an eviction based, rolling recreate strategy, and may not honor `PodDisruptionBudget` for longer than an hour. See [Best Practices]({{< relref "Best Practices" >}}) for information specific to your cloud product.
 
+## Extended Duration Pods in GKE Autopilot (Kubernetes 1.27+)
```

Comment:
Here we can add a link to the GKE doc https://cloud.google.com/kubernetes-engine/docs/how-to/extended-duration-pods